README.md

```diff
@@ -137,15 +135,15 @@ Clone this repository to your desired folder:
       npx stylelint "**/*.{css,scss}" --fix
  
 ### Deployment
-- 1- install `'gh-pages'` with following command:
+- 1- install `gh-pages` with the following command:
 
       npm i -D gh-pages
```

Comment:
- [ ] Good work deploying your application but only one of the links is working(Github pages). Kindly feel free to remove the second deployment on render platform to keep your readme file professional

Reply:
Hi @frank1738  👋,
Thank you so much 🙏  for reviewing my project.
Your nice and constructive feedback is appreciated 🙏.
I have ✔️ and make them fixed.